beard/utils/joblib_helpers.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you rename the file to `joblib.py`? (the `_helpers` suffix is redundant with the name of the module)